lib/angular-evaporate.js

```diff
@@ -113,6 +113,16 @@
             this.contentType = this.file.type || 'binary/octet-stream';
             this.$url = ae.$url + this.name;
 
+            function readableFileSize(size) {
```

Comment:
the function will be recreated with every `forEach` iteration, taking unnecessary memory and process cycles, I propose a different approach later in comments.

and some style guide stuff here) everywhere in the project there are spaces between function names (or `while` keywords) and brackets.